aws/resource_aws_db_instance_test.go

```diff
@@ -3552,6 +3622,55 @@ resource "aws_db_instance" "test" {
 `, rName)
 }
 
+const testAccAWSDBInstanceBaseConfig = `
+resource "aws_db_instance" "test" {
+  allocated_storage       = 10
+  backup_retention_period = 1
+  engine                  = data.aws_rds_orderable_db_instance.test.engine
+  engine_version          = data.aws_rds_orderable_db_instance.test.engine_version
+  instance_class          = data.aws_rds_orderable_db_instance.test.instance_class
+  name                    = "baz"
+  parameter_group_name    = "default.mysql5.6"
```

Comment:
Ugh. This is my doing but we should be using a `aws_rds_engine_version` in this config... Different PR.

Reply:
@YakDriver I gave it a go with the new data source in the latest commit -- looks ok?